timeseries/tests/unittests/models/chronos/test_model.py

```diff
@@ -52,7 +61,14 @@ def chronos_bolt_model(*args, hyperparameters=None, **kwargs):
     return ChronosModel(*args, **kwargs)
 
 
-TESTABLE_MODELS = [ChronosModel, chronos_bolt_model]
+def chronos_bolt_model_with_finetuning(*args, hyperparameters=None, **kwargs):
+    hyperparameters = copy.deepcopy(hyperparameters or {})
+    hyperparameters |= {"model_path": CHRONOS_BOLT_TEST_MODEL_PATH, "fine_tune": True, "fine_tune_steps": 10}
+    kwargs["hyperparameters"] = hyperparameters
+    return ChronosModel(*args, **kwargs)
+
+
+TESTABLE_MODELS = [ChronosModel, chronos_bolt_model, chronos_bolt_model_with_finetuning]
```

Comment:
cool. note also that this doesn't cover chronos classic fine tuning (I think we can skip for now) and also we should still add to smoke tests.